components.py

```diff
@@ -119,18 +120,24 @@ def claim_type_line_chart(df, animated=True):
         "grid": {"right": 140},
         "series": seriesList,
     }
-    st_echarts(options=option, height="450px", width="640px", key="chart")
+    st_echarts(options=option, height="550px", key="chart")
```

Comment:
little nit but can we make the height like 525 or something? it's not lining up with the bottom of the metric bans for me